Regressions/Vehicles/Cessna_172.py

```diff
@@ -178,9 +173,9 @@ def vehicle_setup():
     fuselage.width                              = 42.         * Units.inches     # Width of the fuselage
     fuselage.heights.maximum                    = 62. * Units.inches    # Height of the fuselage
     fuselage.lengths.total                      = 326.         * Units.inches            # Length of the fuselage
-    fuselage.lengths.empennage                  = 161. * Units.inches  
+    fuselage.lengths.tail                       = 161. * Units.inches  
     fuselage.lengths.cabin                      = 105. * Units.inches
-    fuselage.lengths.structure                  = fuselage.lengths.total-fuselage.lengths.empennage 
+    fuselage.lengths.structure                  = fuselage.lengths.total-fuselage.lengths.tail 
```

Comment:
Also here, just to make sure the fuselage length considered is only for the tail